spec/support/capybara.rb

```diff
@@ -9,7 +9,7 @@
   Capybara.default_max_wait_time = 4
   Capybara.javascript_driver = :chrome_en
 
-  port = ENV.fetch('CAPYBARA_SERVER_PORT', '0').to_i
+  port = ENV.fetch('CAPYBARA_SERVER_PORT', ENV.fetch('TEST_ENV_NUMBER', '1').to_i + 3000).to_i
```

Comment:
Can't you set `CAPYBARA_SERVER_PORT` directly to something like 3003?

Reply:
parallel_spec will start threads and the only thing differentiating those threads is the `TEST_ENV_NUMBER` env variable injected by parallel_spec. So we need to use this to dynamically adjust the port used in the feature tests (but I might be wrong).

Reply:
Okay, I would at least then write a comment there to explain why this check is so weird